src/librustc_resolve/build_reduced_graph.rs

```diff
@@ -390,7 +388,7 @@ impl<'b> Resolver<'b> {
                 debug!("(building reduced graph for external crate) building module {} {:?}",
                        name, vis);
                 let parent_link = ModuleParentLink(parent, name);
-                let module = self.new_module(parent_link, Some(def), true);
+                let module = self.new_module(parent_link, Some(def), DUMMY_NODE_ID);
```

Comment:
what does using DUMMY_NODE_ID mean? Seems like it is just ignored when searching, so it would probably be better to use an Option<NodeId> and None here.